### ISSUE TITLE
Tweak the main page

### ISSUE THREAD

**Issue by edakturk14** [BuidlGuidl/batch6.buidlguidl.com#6]
## Goal

Discuss with your peers what you would like the homepage to show! It can be a good opportunity to tweak the app colors/layout etc.

As in #8, keep it simple (no big images, external assets, etc)

This can be an ongoing issue while you all keep implementing things.

**Comment by MayowaObisesan** [comment]
@edakturk14 . Is it a good idea to talk about this issue on the telegram group? Since it says to discuss with my peers.
I'll like to pick up this issue.

**Comment by edakturk14** [comment]
yes def go for it - for example if you have any design suggestions you can mention that in the group and get feedback from the folks. (eg: change of logo, colors, maybe add custom images -- totally up to you)

happy to assign it to you, thanks for taking it on!

**Comment by MayowaObisesan** [comment]
Thanks. I'll follow up in the group based on your reply.

**Comment by wildanvin** [comment]
Hello @MayowaObisesan!

I don't know the progress that you have on the main page but I would like to propose an idea.

A while ago I made a project with Scaffold-Eth 2 and it had an example UI page that was awesome. I spent more time that I should watching it 😅. It is this one:

https://github.com/BuidlGuidl/batch6.buidlguidl.com/assets/48974550/c4d67251-8423-4ee7-8a24-edc5932dde0f

Basically my idea is to make this our main page. We can set the fee to change the message 0.008 ETH and distribute to the memebers of the batch. It would require some tweaks in the contract and the front end but it shouldn't be super hard (hopefully), because it is already implemented

This is the github repo: https://github.com/wildanvin/c-entry
and the page: https://c-entry.vercel.app/example-ui

I don't know if maybe this is too much? would love to hear your feedback
